client/ayon_core/modules/deadline/plugins/publish/submit_publish_cache_job.py

```diff
@@ -344,12 +346,8 @@ def process(self, instance):
 
         deadline_publish_job_id = None
         if submission_type == "deadline":
-            # get default deadline webservice url from deadline module
-            self.deadline_url = instance.context.data["defaultDeadline"]
-            # if custom one is set in instance, use that
-            if instance.data.get("deadlineUrl"):
-                self.deadline_url = instance.data.get("deadlineUrl")
-            assert self.deadline_url, "Requires Deadline Webservice URL"
+            deadline_url = instance.data["deadline"]["url"]
+            assert deadline_url, "Requires Deadline Webservice URL"
```

Comment:
Another bug is `self.deadline_url` is not defined. 
This worked for me.
```suggestion
            self.deadline_url = instance.data["deadline"]["url"]
            assert self.deadline_url, "Requires Deadline Webservice URL"
```

Reply:
Thanks, updated

Reply:
This would be even better if we wouldn't need to store it on `self` on the plug-in because it's shared between instances.

We should really just change any usage of `self.deadline_url` to `instance.data["deadline"]["url"]`  - right?